package-system/squish-ccr/Findsquish-ccr.cmake

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of this distribution.
+# 
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+
+# this file actually ingests the library and defines targets.
+
+set(LIB_NAME "squish-ccr")
+set(TARGET_WITH_NAMESPACE "3rdParty::${LIB_NAME}")
+if (TARGET ${TARGET_WITH_NAMESPACE})
+    return()
+endif()
+
+set(${LIB_NAME}_INCLUDE_DIR ${CMAKE_CURRENT_LIST_DIR}/${LIB_NAME}/include)
+set(${LIB_NAME}_LIBRARY_DIR ${CMAKE_CURRENT_LIST_DIR}/${LIB_NAME}/bin)
+
+add_library(${TARGET_WITH_NAMESPACE} INTERFACE IMPORTED GLOBAL)
```

Comment:
We can make this a STATIC IMPORTED GLOBAL target and avoid the use of target_link_libraries.
Instead we can use IMPORTED_LOCATION

Reply:
Does that apply even though this is a shared library?

Reply:
For a SHARED library we should use SHARED IMPORTED GLOBAL then with the `IMPORTED_IMPLIB`  property on Windows only in addition to the `IMPORTED_LOCATION` property

Reply:
We have an example of setting the properties for the OpenImageIO library: https://github.com/o3de/3p-package-source/blob/52e16f4c574eecf1f9fe02cd79bd48eebe75c38e/package-system/openimageio-opencolorio/distribution/FindOpenImageIO.cmake#L188-L196

Reply:
Updated